internal/pkg/runtime/engine/apptainer/container_linux.go

```diff
@@ -2351,7 +2351,7 @@ func (c *container) createCwdDir(system *mount.System) error {
 	cwdHost := filepath.Clean(c.engine.EngineConfig.GetCwd())
 	if cwdHost == "" || cwdHost == "/" {
 		c.skipCwd = true
-		sylog.Warningf("No current working directory set: skipping mount")
+		sylog.Debugf("No current working directory set: skipping mount")
```

Comment:
I'm not sure it's a good idea to change all these messages to debug level.

The nested apptainer is invoked with `--pwd /` so I'm pretty sure it's only the second case that we care about.  How about it stays as a warning if `cwdHost == ""` but doesn't print anything if `cwdHost == "/"`.

Reply:
thanks for the comment, updated.